os/drivers/audio/alc5658char.c

```diff
@@ -386,7 +386,7 @@ static int alc5658char_configure(FAR struct alc5658char_dev_s *priv, FAR const s
 		break;
 		case AUDIO_FU_MUTE: {
 			/* Set the mute true/false */
-			priv->mute = !!(caps->ac_controls.b[0]);
+			priv->mute = ! !(caps->ac_controls.b[0]);
```

Comment:
need check here.

Reply:
@btheosam please see to this. In my opinion we should remove ! ! .

Reply:
@shivgarg yes, remove one !